aws/templates/codebuild/deploy_cid_checker.yaml

```diff
@@ -0,0 +1,52 @@
+# Do not change version. This is the version of aws buildspec, not the version of your buildspec file.
+version: 0.2
+
+env:
+  variables:
+    PROJECT: 'filecoin'
+    ENVIRONMENT: 'dev'
+    SUB_ENV: 'glif'
+    SHORT_REGION: 'apn1'
+    AWS_ACCOUNT_ID: '499623857295'
+    DOCKERHUB_ORGANIZATION: 'protofire'
+    KUBECTL_VERSION: 1.24.3
+    HELM_VERSION: 3.1.0
+    DOCKERHUB_REPO_NAME: cid-checker
+    APP_VERSION: 0.1.3
+  secrets-manager:
+    DOCKERHUB_USERNAME: "filecoin-mainnet-apn1-glif-dockerhub-glifio:username"
+    DOCKERHUB_PASS: "filecoin-mainnet-apn1-glif-dockerhub-glifio:password"
+    AWS_ACCESS_KEY_ID: "filecoin-dev-apn1-glif-codebuild-user:AWS_ACCESS_KEY_ID"
+    AWS_SECRET_ACCESS_KEY: "filecoin-dev-apn1-glif-codebuild-user:AWS_SECRET_ACCESS_KEY"
+
+phases:
+  install:
+    commands:
+      - echo Environment Variables
+      - printenv
+      - echo Installing kubectl
+      - curl -o kubectl https://amazon-eks.s3.us-west-2.amazonaws.com/${KUBECTL_VERSION}/2021-07-05/bin/linux/amd64/kubectl
+      - chmod +x ./kubectl
+      - mv kubectl /usr/bin/
+      - echo Logging to AWS eks
+      - aws eks --region $AWS_REGION update-kubeconfig --name $PROJECT-$ENVIRONMENT-$SHORT_REGION-$SUB_ENV-eks
+      - echo Installing Helm
+      - curl -o helm https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz
+      - chmod +x ./helm
+      - mv helm /usr/bin
+
+  pre_build:
+    commands:
+      - echo Build STARTED
+      - docker login --username $DOCKERHUB_USERNAME --password $DOCKERHUB_PASS
+  build:
+    commands:
+      - echo Building docker images
+      - make docker_build_all
+      - docker tag cid-checker-frontend:latest $DOCKERHUB_ORGANIZATION/$DOCKERHUB_REPO_NAME:$APP_VERSION-frontend
+      - docker tag cid-checker-api:latest $DOCKERHUB_ORGANIZATION/$DOCKERHUB_REPO_NAME:$APP_VERSION-backend
+      #TODO: docker tag cid-checker-watcher:latest $DOCKERHUB_ORGANIZATION/
+      - docker push $DOCKERHUB_ORGANIZATION/$DOCKERHUB_REPO_NAME:$APP_VERSION-frontend
+      - docker push $DOCKERHUB_ORGANIZATION/$DOCKERHUB_REPO_NAME:$APP_VERSION-backend
+      - cd helm
+      - helm install --upgrade cid-checker . 
```

Comment:
No. Here it's have no connection with dockerhub. It's a helm chart release name, and changing it will affect some other hardcoded places. It's kept hardcoded on purpose